content/concepts/introduction/technical-faq.md

```diff
@@ -0,0 +1,99 @@
+#### What service do transaction validators provide?
+
+Validators determine if transactions meet protocol requirements, and are therefore “valid.” The service validators uniquely provide is grouping transactions into ordered units, agreeing on one such ordering specifically to prevent double spending.
+
+See [The Ripple Ledger Consensus Process](x) and the <a href="https://ripple.com/insights/ripple-labs-tech-talk-consensus-within-the-ripple-protocol/" target="_blank">Ripple Labs Tech Talk: Understanding Consensus <i class="fa fa-external-link" aria-hidden="true"></i></a><!--#{ fix md highlighting_ #}--> for more information about the consensus process.
+
+
+#### How much does it cost to run a validator?
+
+Running a validator does not require any fees or XRP. It is comparable in cost to running an email server in terms of electricity.
+
+
+#### What are Unique Node Lists (UNLs)?
+
+They are the lists of transaction validators a given participant believes will not conspire to defraud them.
+
+
+#### Which UNL should I select?
+
+Since anybody can run a validator, the burden is on the network participants to choose a reliable set. Currently, _Ripple_ (further mentions of Ripple in this document that are italicized to represent the company) provides a default and recommended list which we expand based on watching the history of validators operated by _Ripple_ and third parties. Eventually, _Ripple_ intends to remove itself from this process entirely by having network participants select their own lists based on publicly available data about validator quality.
+<!--#{ TODO: See the references to _Ripple_ the company vs Ripple (non-ital) as what I think is the XRP Ledger. Should we just replace all references to the non-italicized Ripple with "XRP Ledger" or "`rippled`" as appropriate? #}-->
+
+#### If _Ripple_ recommends adoption of its UNL, doesn’t that create a centralized system?
+
+No. The Ripple network is opt-in. Each participant directly or indirectly chooses its UNL. Should _Ripple_ stop operating or should _Ripple_ act maliciously, participants could change their UNLs to continue using the network.
+
+
+#### What is the validator incentive structure for validators not run by _Ripple_?
+
+If the Ripple network becomes successful and is widely used for interbank settlement, there will be an incentive for participants to ensure the reliability and stability of the network. If this happens, institutions will run Ripple servers to participate in the network. Once you are running a server, the additional cost and effort to operate a validator is essentially zero—it would simply involve flipping a software switch from off to on. It is the validators who decide the evolution of the Ripple network, so the primary incentive to run a validator is to preserve and protect the stable operation and sensible evolution of the network.
```

Comment:
"the Ripple network" → "the XRP Ledger" (2x)

"Ripple servers"→"`rippled` servers"